app/services/json_schema/validate_participant_event.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require "json_schema/base"
+
+module JsonSchema
+  class ValidateParticipantEvent < Base
+    def default_config
+      super.merge(
+        {
+          version: "0.2",
+          schema_path: "ecf/participant_declarations",
+        },
+      )
+    end
+  end
```

Comment:
That looks quite different to `ValidateContract`. Is that difference intentional?

Reply:
Yes. You are correct. That should be private and the InitializeWithConfig isn't directly required since it's inheritance will take care of that.

Reply:
And now gone. The Contract and ParticipantEvent are now passed through by the calling service to a lib class to perform the validation